src/test/java/uk/gov/justice/hmiprobation/casesampler/services/SamplePickerTest.kt

```diff
@@ -0,0 +1,105 @@
+package uk.gov.justice.hmiprobation.casesampler.services
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import uk.gov.justice.hmiprobation.casesampler.dto.Case
+import uk.gov.justice.hmiprobation.casesampler.dto.Gender
+import uk.gov.justice.hmiprobation.casesampler.dto.RiskOfSeriousHarmLevel
+import uk.gov.justice.hmiprobation.casesampler.dto.Sample
+import uk.gov.justice.hmiprobation.casesampler.dto.SentenceType.COMMUNITY_SENTENCE
+import uk.gov.justice.hmiprobation.casesampler.dto.SentenceType.POST_CUSTODY
+import uk.gov.justice.hmiprobation.casesampler.dto.Stratum
+import uk.gov.justice.hmiprobation.casesampler.dto.Stratum.FEMALE
+import uk.gov.justice.hmiprobation.casesampler.dto.Stratum.MALE_COMMUNITY_LOW
+import uk.gov.justice.hmiprobation.casesampler.utils.Result
+import uk.gov.justice.hmiprobation.casesampler.utils.Size.SampleSize
+import java.time.LocalDate
+import kotlin.random.Random
+
+class SamplePickerTest {
+
+    lateinit var samplePicker: SamplePicker
+
+    @BeforeEach
+    fun setup() {
+        samplePicker = SamplePicker(Random(1))
```

Comment:
hardcoding seed to make tests deterministic... better ways of doing this I guess